requirements/edx/base.txt

```diff
@@ -71,6 +71,7 @@ path.py==7.2
 piexif==1.0.2
 Pillow==3.1.1
 polib==1.0.3
+py2neo==3.1.1
```

Comment:
No rule-of-thumb is set AFAIK. But, for this, it is not critical-path for running the application; it is only needed for this management command. So I thought breaking it out separately made sense.